core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/concat.kt

```diff
@@ -40,6 +43,64 @@ public fun <T> DataFrame<T>.concat(frames: Iterable<DataFrame<T>>): DataFrame<T>
 
 public fun <T, G> GroupBy<T, G>.concat(): DataFrame<G> = groups.concat()
 
+/**
```

Comment:
Really nice and helpful kdoc :) Maybe something similar could be put on the documentation website?

Reply:
For sure, we will put it on website!